modules/oc-adm-by-example-content.adoc

```diff
@@ -365,16 +376,45 @@ Watch platform certificates.
 == oc adm ocp-certificates regenerate-leaf
 Regenerate client and serving certificates of an OpenShift cluster
 
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate all known leaf certificates on the cluster.
+  oc adm certificates regenerate-leaf -A --all
+
+  # Regenerate a leaf certificate contained in a particular secret.
+  oc adm ocp-certificates regenerate-leaf -n openshift-config-managed kube-controller-manager-client-cert
+----
+
 
 
 == oc adm ocp-certificates regenerate-machine-config-server-serving-cert
 Regenerate the machine config operator certificates in an OpenShift cluster
 
+.Example usage
+[source,bash,options="nowrap"]
+----
+  # Regenerate the MCO certs without modifying user-data secrets
+  oc adm certificates regenerate-machine-config-server-serving-cert --update-ignition=false
```

Comment:
the command should be `oc adm ocp-certificates regenerate-machine-config-server-serving-cert --update-ignition=false`